code/atom.dm

```diff
@@ -433,6 +437,13 @@
 				T2.neighcheckinghasproximity++
 		if(src.opacity)
 			T.opaque_atom_count++
+		if(src.bound_width == 32 && src.bound_width == src.bound_height)
```

Comment:
can a 32x32 overlay not cover multiple turfs?

Reply:
I am pretty sure the bounding box cannot be on two tiles at once if it's the size of one. We don't have pixel movement turned on.

In discord leah brought up values <32, I am not totally sure but it would make little sense to have values smaller than 32 because we don't use pixel movement. May change to `<=`